test/snippet/search/dream_index/interleaved_bloom_filter_constructor_compressed.cpp

```diff
@@ -0,0 +1,12 @@
+#include <seqan3/search/dream_index/interleaved_bloom_filter.hpp>
+
+int main()
+{
+    // Construct an uncompressed Interleaved Bloom Filter.
+    seqan3::interleaved_bloom_filter ibf{43, 8192, 3};
+
+    // Fill `ibf` with data.
```

Comment:
Should IBF not be capitalized?

Reply:
It's the variable name. It comment should indicate that you do something with the uncompressed IBF and then compress it.